src/effect/fadeInDown.js

```diff
@@ -14,16 +14,14 @@ export default {
     'panel': {
         'position': 'fixed',
         'backgroundColor': '#fff',
-        'width': '400px',
-        'height': '300px',
+        'width': 'auto',
+        'height': 'auto',
         'top': '50%',
         'left': '50%',
-        'marginTop': '-150px',
-        'marginLeft': '-200px',
         'borderRadius': '5px',
         'boxSizing': 'border-box',
         'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.3)',
-        'transform': 'translate3d(0, 0, 0)',
+        'transform': 'translate3d(-50%, -50%, 0)',
```

Comment:
When the dialog animates, it based on transform property.
For example in the case of 'fadeInDown',  it moves from (0, -100px, 0) to (-50%, -50%, 0).
So, I don't want you to use transform property for centering the dialog.